test/Garnet.test/Extensions/RateLimiterTxn.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Text;
+using Garnet.common;
+using Garnet.server;
+using Tsavorite.core;
+
+namespace Garnet
+{
+    sealed class RateLimiterTxn : CustomTransactionProcedure
+    {
+        public override bool Prepare<TGarnetReadApi>(TGarnetReadApi api, ref CustomProcedureInput procInput)
```

Comment:
Could you help me understand the difference between CustomProcedureInput and ArgSlice input. Based on docs I could use ArgSlice.

Reply:
That changed as part of a recent refactoring.
Good point. Will update the documentation as well.